### ISSUE TITLE
Possible error in the documentation of DifferentialActionModelAbstractTpl

### ISSUE THREAD

**Issue by lwygzh** [loco-3d/crocoddyl#1183]
Hi,

I was reading the documentation of [`DifferentialActionModelAbstractTpl`](https://gepettoweb.laas.fr/doc/loco-3d/crocoddyl/master/doxygen-html/classcrocoddyl_1_1DifferentialActionModelAbstractTpl.html#:~:text=template%3Ctypename%20_Scalar%3E%0Aclass%20crocoddyl%3A%3ADifferentialActionModelAbstractTpl%3C%20_Scalar%20%3E) and I think there might be an error within the documentation. When showing what the cost function for `DifferentialActionModelAbstractTpl` should describe, it shows that

```math
l(x, u) = \int_{0}^{\delta_t}{a(r(x, u))}dt
```

I was wondering if the value of `l(x, u)` in this case should actually be

```math
l(x, u) = a(r(x, u))
```

**Comment by cmastalli** [comment]
Yes, you're right! We integrate the cost function in the integrated action. I would be fantastic if you can help us to improve the documentation.

**Comment by lwygzh** [comment]
Of course, I would be glad to help! I can do a PR with the updated doc.
